index.html

```diff
@@ -45,34 +46,34 @@
               <a class="navigation__link" href="#">Contact Us</a>
             </li>
           </ul>
-        </div>
+        </nav>
 
-        <button>Donate Now</button>
+        <button class="red-button">Donate Now</button>
 
-    </div>
+      </header>
 
-    <div class="content">
+    <main class="content">
         <div class="main">
```

Comment:
Consider using different [semantic tags](https://www.w3schools.com/html/html5_semantic_elements.asp). Fallback to `div`s when absolutely necessary - quite often there is a better alternative.